cfgov/tccp/jinja2/tccp/card.html

```diff
@@ -187,7 +194,7 @@ <h2>
                     {{ apr(card.purchase_apr_median) }}
                 {% endif %}
             </div>
-            <div class="h4 u-mt0">
+            <div class="h4 u-mt0" role="term" aria-details="card-purchase-apr card-purchase-apr-definition">
```

Comment:
This causes an issue when running Lighthouse in Chrome for me:

<img width="726" alt="image" src="https://github.com/cfpb/consumerfinance.gov/assets/654645/eb3fbe56-3e0b-40b4-b9c5-938d8c8c6187">

That link points [here](https://dequeuniversity.com/rules/axe/4.8/aria-valid-attr-value). The ARIA spec [says](https://www.w3.org/TR/wai-aria/#aria-details):

> The aria-details attribute references a single element 

but MDN [says](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-details):

> The global aria-details attribute identifies the element (or elements)
> The aria-details attribute takes the id, or space separated list of ids of the elements

I wonder if Lighthouse is complaining because it assumes this should only be a single element?

Reply:
Even the example on the MDN page includes multiple ids: `aria-details="bezier bezImg"`. And the lighthouse [error explanation](https://dequeuniversity.com/rules/axe/4.8/aria-valid-attr-value) specifically says an acceptable value is `A list of one or more ID references`. Soooo I'm going to dig into the [axe rule](https://github.com/GoogleChrome/lighthouse/blob/a511e5899e90dd9b943b85901fa3c40024d6a5f1/core/audits/accessibility/aria-valid-attr-value.js) and see if it's a bug.